projects/storefrontlib/src/cms-components/user/login-form/login-form.component.ts

```diff
@@ -37,8 +33,14 @@ export class LoginFormComponent implements OnInit, OnDestroy {
   ) {}
 
   ngOnInit(): void {
+    const routeState = this.winRef.nativeWindow?.history?.state;
+    const prefilledEmail = routeState?.['newUid'];
```

Comment:
Looking at the previous code, it seems this was deprecated. Not sure about why. Are we sure on keeping this here?

Reply:
If we keep it, can we get the `newUid` from the `activatedRoute.snapshot`?

Reply:
Let's make a new ticket out of it, since refactoring `login.component` isn't really a part of this PR

Reply:
sounds fair